docs/source/usage.rst

```diff
@@ -74,3 +74,58 @@ Positions:
   - ``H``: OK
   - ``7``: failure to charge HV capacitor
 
+BLE - Bluetooth® Low Energy
+###########################
+
+BLE Heart Rate Service
+----------------------
+
+The MultiGeiger provides a Bluetooth® Low Energy (BLE) service to allow the collection of the Geiger-Mueller count rate via a `GATT Heart Rate Service <https://www.bluetooth.com/specifications/gatt/services/>`_ (Service UUID 0x180D). The following characteristics are used:
+
+- 0x2A37 ('Heart Rate Measurement Characteristic'):
+
+  - The first byte is a collection of status flags, according to the service's standard
+  - 'Heart Rate Measurement' as 16 bit value (little endian), corresponds to Geiger-Mueller counts per minute (CPM)
+  - 'Energy Expenditure' as 16 bit value (little endian), represents a rolling packet counter
+- 0x2A38 ('Heart Rate Sensor Position Characteristic')
+
+  - 'Sensor Position' as 8 bit value, corresponds to TUBE-TYPE, allowing the conversion of CPM to radiation rate
+- 0x2A39 ('Heart Rate Control Point Characteristic')
+
+  - Write characteristic, required by service's standard to reset Energy Expenditure to 0. Writing 0x01 resets the rolling packet counter to 0.
+
+Testing BLE
+-----------
+
+Any decent Bluetooth® scanning app should be able to connect to the device and show the notification values.
+Known to work well is the nordic® nRF Connect app, which should be available for iOS and Android.
+More infos about the Android app can be found `here on github <https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-mobile>`_.
```

Comment:
cool, that works!

Reply:
Right, they provide also the .APK directly.
Glad to hear it works :)